docs/FAQ.md

```diff
@@ -97,7 +97,7 @@ Some use cases:
 
 - Show/hide some inputs if another input has a value
 - Show/hide some inputs if another input has a specific value
-- Show/hide some inputs if the current form values matches specific constraints
+- Show/hide some inputs if the current form value matches specific constraints
```

Comment:
```suggestion
- Show/hide some inputs if the current form value match specific constraints
```

Reply:
Thanks for your quick response! But I don't understand the suggested change.... 😢It seems `matches` is  right since `the current form value` is singular.
Or.. if you have any reason to stick on `match`?  (I just thought in a grammar way, so I thought maybe you have more intention in it.)

Reply:
Never mind, I inverted both lines while reviewing...